lib/bootloader_setup.pm

```diff
@@ -212,6 +212,11 @@ sub specific_bootmenu_params {
         }
     }
 
+    if (check_var('ARCH', 'aarch64') && check_var('DISTRI', 'sle')) {
+        $args .= " plymouth=0 linuxrc.debug=trace";
+        record_soft_failure "bsc#1005313 - Unhandled level 1 translation fault";
+    }
```

Comment:
yes, this is obviously a good idea. But as we have many more open bugs with "problems on bootup" I am wondering if we can't find a good permanent approach but not changing the default, e.g. to be run in parallel. I'll show what I mean.